docs/content/references/sui-api/graphql/guides.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: Guides
```

Comment:
This title could be better. Maybe from your desc: Sui GraphQL Guidelines?

Reply:
I think the title is ok. If we do `Sui GraphQL Guidelines` it will appear like that on the left, and it's becoming too much. Folks are in the Sui docs, so we shouldn't extra annotate things with Sui, in my opinion.